config/rukpak/apis/webhooks/resources/deployment.yaml

```diff
@@ -13,9 +13,17 @@ spec:
       labels:
         app: webhooks
     spec:
+      securityContext:
```

Comment:
Is spec.securityContext a valid DeploymentSpec field:

```bash
$ k explain deploy.spec | grep securityContext
```

Returns nothing. But deploy.spec.template.spec does appear to have a securityContext field.

Reply:
This `securityContext` is already defined inside of the `deploy.spec.template.spec` but it is hard to see because of how GitHub is rendering it. You can hit the little arrow button above this to see.

Reply:
Yeah my bad. What's even worse is that I pressed the drop down, and still mixed this up.

/approve
/lgtm